homeassistant/components/remote/harmony.py

```diff
@@ -26,12 +26,13 @@
 
 DEFAULT_PORT = 5222
 DEVICES = []
```

Comment:
Both `DEFAULT_PORT` and `DEVICES` are still used. `DEVICES` is used later on in existing code.